bin/pulsar

```diff
@@ -221,7 +221,6 @@ fi
 
 if [ -z "$PULSAR_GLOBAL_ZK_CONF" ]; then
     PULSAR_GLOBAL_ZK_CONF=$DEFAULT_GLOBAL_ZK_CONF
-    PULSAR_CONFIGURATION_STORE_CONF=$DEFAULT_GLOBAL_ZK_CONF
```

Comment:
This seems to be unrelated with the PR

Reply:
> While running `bin/pulsar configuration-store` the script overrides the custom conf file with default if `PULSAR_GLOBAL_ZK_CONF` is not provided.

This change fixes this ^^

For example, try running 
```
export PULSAR_CONFIGURATION_STORE_CONF=<custom path>
bin/pulsar configuration-store
```

The DEFAULT_GLOBAL_ZK_CONF will be picked up instead of your custom file.